hack/bump-cf-for-k8s.sh

```diff
@@ -10,3 +10,7 @@ pushd "${CF_FOR_K8s_DIR}"
   vendir sync -d config/metrics/_ytt_lib/metric-proxy="${REPO_DIR}/config"
 popd
 
+# sed -i "s|version:.*|version: dev-test|" $REPO_DIR/config/values.yml
+
+# On Mac
```

Comment:
Can you give a short description of what this is doing? Also, did you mean to leave the comment above?

Reply:
As in add it to the code as well?

This is modifying the `config/values.yml` to ensure that the version number is filled out during testing. I didn't mean to commit `version: dev-test` within the file itself. It actually should be blank.

I was seeing errors when this wasn't filled and resulted in some type of error. I can't remember what it was or maybe I was conflating errors.

Reply:
What I meant was adding "This is modifying the config/values.yml to ensure that the version number is filled out during testing." as a comment in the code in addition to #On Mac.

I think you're right that we do need it when running locally, although, maybe we can just leave it blank for now and add it to the script as a separate commit.

Reply:
Just saw your last comment 👍